content/english/Kusto-KC7/activity-4.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Activity 4: Catch the Hacker"
+draft: false
+weight: 16
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/HjKkQVH5-rQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+{{< notice note >}}This is a much shorter section, we're just going to explain the scenario and let you go for free hunting time. You can take however much time you'd like to on this portion, but for live sessions we generally give participants 30-40 minutes before moving on.
+
+- **Be sure you use the SecurityLogs database**
+- **Submit your answers "Introducing the Hackers" challenge on the Scoreboard**
+
+#### **Happy Hunting!**{{< /notice >}}
+
+### Introducing the Hackers
+
+
+Now that you’ve completed your initial round of training, you are ready to work your first case in the SOC!
+
+A security researcher tweeted that the domain *“immune.tech”* was being used by hackers. Apparently the hackers are sending this domain inside credential phishing emails.
+
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/CTF1.png?raw=true" alt= “CTF1” width="value" height="value">
+
+{{< alert theme="warning" >}}
+⚠️NOTE! This domain and others encountered in this game are fictional and are not representative of actual malicious activity. It is a commonly used safety protocol to use the format immune[.]tech or domain[.]com when mentioning potentially malicious links online. Adding these brackets around the dot prevents the creation of a clickable link. When searching logs, be sure to remove the brackets from the domain name. {{< /alert >}}
```

Comment:
Let's move the first part of this note to Tutorial 2.

Reply:
Done.